### PR TITLE
Pay Invoice Support

### PR DESCRIPTION
**Pull request by gasparmedina**
The possibility of making bill payments in 3 places was added.
Payment from a channel, fast payment and token delivery.

This pull requests depends on https://github.com/rsksmart/lumino/pull/10

### CODE REVIEW

File: src/actions/paymentsActions.js [hunk 1]
```diff
@@ -135,6 +135,7 @@ export const quickPayment = (partnerAddress, tokenAddress, totalDeposit) => asyn
                 )
             })
             .then(response => {
+                console.log(JSON.stringify(response));
```
Comment (anarancio) [inline src/actions/paymentsActions.js]: Remove this console.log

File: src/actions/paymentsActions.js [hunk 2]
```diff
@@ -152,6 +153,33 @@ export const quickPayment = (partnerAddress, tokenAddress, totalDeposit) => asyn
...
+export const invoicePayment = (invoice) => async (dispatch,
+    getState) => {
+    await getTokenApp("invoicePayment");
+    new Promise((resolve, reject) =>
+        client
+            .post(`/api/v1/paymentsLumino/invoice`, {
+                coded_invoice: invoice
+            })
+            .then(response => {
+                console.log(JSON.stringify(response));
```
Comment (anarancio) [inline src/actions/paymentsActions.js]: Remove this console.log

File: public/static/endpointConfig.js [hunk 3]
```diff
@@ -1,4 +1,4 @@
-const backendUrl='http://localhost:5001';
```
Comment (marcosmartinez7) [inline public/static/endpointConfig.js:1]: add to .gitignore

File: src/actions/paymentsActions.js (hunk 1 above)
Comment (gasparmedina) [reply src/actions/paymentsActions.js]: ok

File: src/actions/paymentsActions.js (hunk 2 above)
Comment (gasparmedina) [reply src/actions/paymentsActions.js]: ok

File: public/static/endpointConfig.js (hunk 3 above)
Comment (gasparmedina) [reply public/static/endpointConfig.js:1]: ok

**Pull request merged**